monitoring_alerts.tf

```diff
@@ -10,6 +10,32 @@ resource "google_monitoring_notification_channel" "slack_alert" {
   }
 }
 
+resource "google_monitoring_uptime_check_config" "http" {
+  project      = "${var.stackdriver_workspace}"
+  display_name = "${var.uptime_check_host}"
+  timeout      = "5s"
+  period       = "60s"
+
+  http_check {
+    path    = "/status"
+    port    = 443
+    use_ssl = true
+  }
+
+  monitored_resource {
+    type = "uptime_url"
+
+    labels = {
+      project_id = "${var.stackdriver_workspace}"
```

Comment:
This isn't needed and I think we should remove it.

Reply:
Ok, the Terraform doc isn't clear and it's in their example. Removed.